Report.md

```diff
@@ -0,0 +1,113 @@
+# HW3 - GAME OF LIFE EVALUATION
+## REPORT
+*Languages considered- Rust, Fortran and Go*
+
+**Methods:**
+
+Web based user friendly approach used for the human trials.
+User was provided with token via email/slack during the start of the session, and asked to log in to the web page.
+The 1st page was a pre-debugging survey, to gauge the user comfort with debugging the code in various languages. This survey was a rating, on a scale from 1-5. This scale is useful and convenient for evaluation of the 3 languages, and form a weighing metric in the evaluation formula.
+The next page, consisted of instructions to the user, the buggy code and link to ide for debugging in all the 3 languages. Prior to experiment evaluation on the difficulty level of the bugs were performed and care was taken such that most of the users could complete debugging in all 3 languages in the given time slot. 1 logical bug per coding language of similar nature was introduced.
+This page also had a start and stop button to note down the debugging time taken as an evaluation metric. 
+
+
+**Materials:**
+
+*User Material -*
+
+1) bare minimal- just a link to the web page, self-driven and 95% automated. Only the token needs to be provided manually by the admin.
+2) Stable internet connection.
+
+*Admin Materials-*
+
+1) Maintain the web server and client, and a database where the evaluation data is collected.
+2) A script to fetch the metrics and evaluate languages and compare the 3 languages.
+3) AWS server keys and service of the server.
+4) Access to online IDE for every language.
```

Comment:
Should we include that  the admin needs to input some flags when the user was unable to solve some bugs? Or should it be a challenge that could not automated?